quantinuum-hugr/src/lib.rs

```diff
@@ -135,7 +135,6 @@
 //! ```
 //!
 
-#![warn(missing_docs)]
```

Comment:
Why is this changing?

Reply:
Now it's on the workspace `Cargo.toml` (under `[workspace.lints.rust]`).